src/sotauptaneclient.cc

```diff
@@ -14,12 +16,12 @@
 #include "utilities/utils.h"
 
 SotaUptaneClient::SotaUptaneClient(Config &config_in, std::shared_ptr<event::Channel> events_channel_in,
-                                   Uptane::Repository &repo, const std::shared_ptr<INvStorage> storage_in,
+                                   Uptane::Repository &repo, std::shared_ptr<INvStorage> storage_in,
                                    HttpInterface &http_client)
     : config(config_in),
-      events_channel(events_channel_in),
+      events_channel(std::move(std::move(events_channel_in))),
```

Comment:
I know this is already merged, but is this really intentional to have a nested `std::move`? I've never seen that before!